test/C/src/docker/federated/DockerOptions.lf

```diff
@@ -0,0 +1,23 @@
+target C {
+  logging: DEBUG,
+  timeout: 1 s,
+  coordination: centralized,
+  docker: {
+    rti-image: "rti:local",
+    pre-build-script: "foo.sh",
+    pre-run-script: "bar.sh",
+    post-build-script: "baz.sh",
+    no-build: false
+  }
```

Comment:
AFAICT this does not actually function as a test because it will not cause the test suite to fail if the provided scripts are not executed. I am OK with this if this solution is not necessarily long-term.

Reply:
An error would result if the scripts weren't accessible in the container, but it would be better if these scripts were to do something like set an environment variable that would be checked. Could you take a stab at turning this into a more meaningful test, @petervdonovan?

Reply:
Done

Reply:
[279b00e](https://github.com/lf-lang/lingua-franca/pull/2234/commits/279b00ea468dd214212aaf86d66fdee728820f74)